src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -263,7 +262,20 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
      * @return {void}
      */
     fetchAccessStats(): void {
-        const { api, fileId }: Props = this.props;
+        const { api, fileId, hasAccessStats }: Props = this.props;
+        const { isLoadingAccessStats } = this.state;
+        if (!hasAccessStats) {
```

Comment:
Do we need to worry about a scenario like the API call is in flight, and then `hasAccessStats` flips to `false`, causing two setStates, first `accessStats: undefined` then the result of `fetchAccessStatsSuccessCallback`?